src/span-data.js

```diff
@@ -94,6 +94,10 @@ function SpanData(agent, trace, name, parentSpanId, isRoot, skipFrames) {
  * @returns {SpanData} The new child trace span data.
  */
 SpanData.prototype.createChildSpanData = function(name, skipFrames) {
+  if (this.span.isClosed()) {
```

Comment:
Ah sorry, I misremembered and believed `WARN` to log by default. This is definitely fine if off by default.